src/m365/pp/base/PowerPlatformCommand.ts

```diff
@@ -0,0 +1,7 @@
+import Command from '../../../Command';
```

Comment:
Let's put the base class in the m365/base folder along with other command base classes so that we have them all in the same place